code/modules/mob/living/carbon/human/human_abilities.dm

```diff
@@ -230,29 +230,14 @@ CULT
 	if(!can_deploy_droppod(T))
 		return
 
-	var/list/list_of_techs = list()
-
-	for(var/i in GLOB.unlocked_droppod_techs)
-		var/datum/tech/tech_to_use = i
-		list_of_techs += list("[tech_to_use.name]" = tech_to_use)
-
-	if(!list_of_techs.len)
-		to_chat(H, SPAN_WARNING("No droppods currently available."))
-		return
-
-	var/input = tgui_input_list(H, "Choose a tech to deploy at this location", "Tech deployment", list_of_techs)
+	to_chat(H, SPAN_WARNING("No droppods currently available."))
+	return
```

Comment:
Imo you should also remove the ability from RTO stuff (if I didn't notice it being removed elsewhere then whoops)